docs/source/guide/index.rst

```diff
@@ -914,6 +914,90 @@ This command adds the provided artifact to the TUF Metadata using the RSTUF REST
     ╰──────────────────────────────────────────────────────────────────────────────────────────────────╯
 
 
+.. rstuf-cli-artifact-repository
+
+Artifact Addition (``repository``)
+---------------------------
+
+This command provides artifact repository management for the RSTUF repository config.
+
+.. code::
+
+    ❯ rstuf artifact repository --help
+
+    Usage: rstuf artifact repository [OPTIONS] COMMAND [ARGS]...
+
+    Repository management.
+
+    ╭─ Options ─────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╮
+    │ --help  -h    Show this message and exit.                                                                                                                                 │
+    ╰───────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯
+    ╭─ Commands ────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╮
+    │ delete                           Delete repository.                                                                                                                       │
+    │ set                              Set a new repository.                                                                                                                    │
+    │ show                             List configured repositories.                                                                                                            │
+    │ update                           Update repository.                                                                                                                       │
+    │ use                              Switch current repository.                                                                                                               │
+    ╰───────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯
```

Comment:
Nit: Copy and paste it from a smaller (length) terminal. It will have less space on the left.


I would rename the commands here
`set` -> `add` 
It will be aligned with other subcomands, and it sounds more specific.

`use` -> `set`
to set the current repository